integration-tests/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build status-go in a Go builder container
+FROM ghcr.io/foundry-rs/foundry:latest as checkout
+
+ARG github_repo
+ARG github_org
+
+RUN apk update && \
+    apk add git
+
+WORKDIR /app
+
+RUN git clone https://github.com/$github_org/$github_repo
+
+WORKDIR /app/$github_repo
+
+RUN git submodule deinit --force .
+RUN git submodule update --init --recursive
```

Comment:
Won't this just create a lot of very big Docker images? Why can't we do this at image runtime?